### ISSUE TITLE
Meta-refresh inconsistencies with ACT

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3496]
Axe-core's meta-refresh rule is inaccurate in two ways:

1. It should account for the 20-hour timeout. Anything timeout more than 20 hours should pass the rule
2. Axe's method of determining the duration is far more permissive than browsers are, which results in it failing elements that do not redirect at all.

[Passed example 3](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc659a/b5ca868de7980f6944142ecdb849f47ad2cdfb5c.html)

```html
<meta http-equiv="refresh" content="72001; https://w3.org" />
```

[Inapplicable Example 3](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc659a/a943b8cfaefd7d7b16fefc36bc24eed2ce89a4e8.html)

```html
<meta http-equiv="refresh" content="0: https://w3.org" />
```

[Inapplicable Example 4](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc659a/e6e51895cab42ee6adf2ade5b654808d830e673b.html)

```html
<meta http-equiv="refresh" content="-00.12 foo" />
```

[Inapplicable Example 5](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc659a/ab5d8aeef0246a383e782e44744d3989e72fd573.html)

```html
<meta http-equiv="refresh" content="; 30" />
```

[Inapplicable Example 7](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc659a/e95e086753e09bfc8c7c20ab5c3241aef0ad8cfd.html)

```html
<meta http-equiv="refresh" content="+5; https://w3.org" />
```

[Inapplicable Example 8](https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases/bc659a/0dcdacefc963adfbb2d3493e5468f8f12f0ac6de.html)

```html
<meta http-equiv="refresh" content="foo; URL='https://w3.org'" />
```

**Comment by padmavemulapati** [comment]
Validated with latest develop branch code base,
`meta-refresh-no-exceptions` rule for pass and fail cases:
```
<head>
    <meta charset="utf8" />
    <title>Meta-refresh-no-exceptions fail 1</title>
    <meta http-equiv="refresh" content="72001: https://w3.org" />
    <meta http-equiv="refresh" content="90000; https://deque.com/">
    
    <link
      rel="stylesheet"
      type="text/css"
      href="/node_modules/mocha/mocha.css"
    />
```
<img width="1581" alt="image" src="https://user-images.githubusercontent.com/47409579/187353568-8c2062f1-36e3-4539-83cc-d018d4f75c5f.png">
